lib/smart_answer/calculators/state_pension_topup_calculator.rb

```diff
@@ -1,16 +1,35 @@
 module SmartAnswer::Calculators
   class StatePensionTopupCalculator
+    include ActiveModel::Model
+
     FEMALE_YOUNGEST_DOB = Date.parse('1953-04-05')
     MALE_YOUNGEST_DOB = Date.parse('1951-04-05')
     TOPUP_START_DATE = Date.parse('2015-10-12')
     TOPUP_END_DATE = Date.parse('2017-04-05')
     FEMALE_RETIREMENT_AGE = 62
     MALE_RETIREMENT_AGE = 65
 
-    def lump_sum_and_age(dob, weekly_amount, gender)
-      return [] if too_young?(dob, gender)
+    attr_accessor :date_of_birth
+    attr_accessor :gender
+    attr_accessor :weekly_amount
+
+    def initialize(attributes = {})
+      super
+      @gender ||= 'female'
```

Comment:
Yes. The original behaviour is pretty odd as I noted ages ago in this [Trello card][1], but I wanted to limit the scope of the changes I was making. If I can find the relevant commit(s), I'll update the commit message(s) to make things clearer.

[1]: https://trello.com/c/jv765Pwc